prepare-data-python/README.md

```diff
@@ -0,0 +1,83 @@
+
+![image of completed project](images/visualizations.png)
+
+## Introduction
+
+In this section, you'll review an [introduction](https://docs.microsoft.com/learn/modules/data-manipulate-clean/1-introduction?WT.mc_id=academic-55190-ornella) and make sure that your data science environment is set up correctly before continuing on to the next part of the workshop.
+
+## Explore DataFrame information
+
+Next, you will learn how to use Python libraries to [explore an iconic dataset](https://docs.microsoft.com/learn/modules/data-manipulate-clean/2-explore-dataframes?WT.mc_id=academic-55190-ornella). You will be able to understand how to use pandas DataFrames to get an immediate idea about the size, shape, and content of a particular dataset. 
+
+## Work with missing data
+
+Now that you know how to get an overall sense of the dataset you are working with, you will learn [how to identify and  deal with missing values](https://docs.microsoft.com/learn/modules/data-manipulate-clean/3-missing-data?WT.mc_id=academic-55190-ornella).
+
+## Remove duplicate data
+
+Another common thing you'll have to do with most datasets you encounter is [remove duplicate data](https://docs.microsoft.com/learn/modules/data-manipulate-clean/4-duplicate-data?WT.mc_id=academic-55190-ornella). In this section of the workshop, you will learn how to use pandas to detect and remove duplicate entries.
+
+## Combine datasets
+
+Sometimes, you will need to combine datasets together. Luckily, there are several methods available in pandas to [merge and join datasets](https://docs.microsoft.com/learn/modules/data-manipulate-clean/5-combine-datasets?WT.mc_id=academic-55190-ornella).
+
+## Exploratory statistics and visualization
+
+So far, you've learned how to use pandas methods to examine some aspects of a DataFrame, and fill in, remove, and combine data. The final way we will seek to understand our data is by [creating visualizations](https://docs.microsoft.com/learn/modules/data-manipulate-clean/6-exploratory-statistics-visualization?WT.mc_id=academic-55190-ornella).
+
+## Next steps
+
+- [Explore and analyze data with Python](https://docs.microsoft.com/learn/modules/explore-analyze-data-with-python/?WT.mc_id=academic-55190-ornella)
+- [Introduction to machine learning
+](https://docs.microsoft.com/learn/modules/introduction-to-machine-learning/?WT.mc_id=academic-55190-ornella)
+- [Discover the role of Python in space exploration](https://docs.microsoft.com/learn/paths/introduction-python-space-exploration-nasa/?WT.mc_id=academic-55190-ornella)
+
+## Optional Transfer Knowledge activity
```

Comment:
Let's call this "Practice" (I've standardized the terminology to make it more friendly)

Reply:
fixed!